rts/motoko-rts/src/continuation_table.rs

```diff
@@ -144,7 +155,7 @@ pub unsafe fn recall_continuation(idx: u32) -> Value {
 
     let ptr = table.get(idx);
 
-    table.set_scalar(idx, Value::from_scalar(FREE_SLOT));
+    table.set(idx, Value::from_scalar(FREE_SLOT), mem);
```

Comment:
Another untagged scalar

Reply:
Probably benign since it should never be referenced from stable variable data (and thus pose challenges for graph copy).

Reply:
I agree, this seems safe, although internally not nice that tagging is not consistently applied. We can do this later.